src/FSharp.Markdown/MarkdownParser.fs

```diff
@@ -449,6 +449,35 @@ let rec (|ListItems|_|) prevSimple = function
 
 // Code for parsing pipe tables
 
+// Splits table row into deliminated parts escaping inline code and latex
+let rec PipeTableFindSplits (delim : char array) (line : char list) = 
```

Comment:
Its more about the code-style: The function is fine but please call it "pipeTableFindSplits" instead of "PipeTableFindSplits". Only active pattern are a exception here.